<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QMainWindow" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="scanPage">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="video">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="MultipartUrStatus" name="multipartUrStatus" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="resultPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="resultTextBrowser">
          <property name="acceptRichText">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newScanPushButton">
          <property name="text">
           <string>New scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="cameraComboBox"/>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStartCamera">
   <property name="text">
    <string>Start Camera</string>
   </property>
  </action>
  <action name="actionStopCamera">
   <property name="text">
    <string>Stop Camera</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultipartUrStatus</class>
   <extends>QWidget</extends>
   <header>multiparturstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>record()</slot>
  <slot>pause()</slot>
  <slot>stop()</slot>
  <slot>enablePreview(bool)</slot>
  <slot>configureCaptureSettings()</slot>
  <slot>takeImage()</slot>
  <slot>startCamera()</slot>
  <slot>toggleLock()</slot>
  <slot>setMuted(bool)</slot>
  <slot>stopCamera()</slot>
  <slot>setExposureCompensation(int)</slot>
 </slots>
</ui>
